ToolsForCategoricalTowers/gap/Tools.gd

```diff
@@ -241,8 +241,8 @@ DeclareOperation( "MorphismBetweenCoproductsWithGivenCoproducts",
 #! * <A>diagram_S</A> is a list of objects $(A_i)_{i = 0 \dots m - 1}$,
 #! * <A>diagram_T</A> is a list of objects $(B_j)_{j = 0 \dots n - 1}$,
 #! * <A>F</A> is a pair consisting of a list $f = (f_j)_{j = 0 \dots n - 1}$
-#!   of length $m$ of integers between $0, \ldots, m - 1$
-#!   and a list of morphisms $( B_j \rightarrow A_{f_j} )_{j = 0 \dots n - 1}$.
```

Comment:
Sorry, I am not used to reading code in here:
This change is wrong, the other one is correct.

Reply:
I will take care of it.

Reply:
I think the misunderstanding is happening because Github _incorrectly_ shows this change as if it were in the documentation of `MorphismBetweenCoproductsWithGivenCoproducts`. Both changes are in the documentation of `MorphismBetweenDirectProducts(WithGivenDirectProducts)`.

Reply:
Yes, GitHub confuses me :)